tests/http_bench.ts

```diff
@@ -23,12 +23,21 @@ async function handle(conn: Deno.Conn): Promise<void> {
   }
 }
 
-async function main(): Promise<void> {
+function acceptRoutine(): void {
+  listener
+    .accept()
+    .then(
```

Comment:
Honestly I cannot see difference for this patch (especially after the V8 async function spec update). Essentially we are still waiting for the previous `accept` to complete before processing using `handle(conn)`, and before the next `accept` takes place...

Also I believe queuing like what we did in `http` does not make sense here. The queuing + `acceptRoutine` part in `http` serves more of a hack to allow async iterator to work, while also allowing user to plug their (potentially blocking) handling without stalling future greedy connection accepts -- neither of which is the case here...

Reply:
Sure. Thank you.